backend/pom.xml

```diff
@@ -53,12 +48,6 @@
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-test</artifactId>
             <scope>test</scope>
-            <exclusions>
-                <exclusion>
-                    <groupId>org.junit.vintage</groupId>
-                    <artifactId>junit-vintage-engine</artifactId>
-                </exclusion>
-            </exclusions>
```

Comment:
This is going to be added back, did I read that in the chat? Seeing it removed worries me a little because presumably the Spring Boot team had a good reason for adding it.

Reply:
When I pasted that back locally to re-test I accidentally put it under the `junit` dependency (an "Adam revert") which worked fine. Now that I reverted it back to being under `spring-boot-starter-test`, I get still see `Tests run: 0, Failures: 0, Errors: 0, Skipped: 0` when executing a `mvn test` through NetBeans.

Reply:
I re-added it back given our findings in the chat that we're not using JUnit 5 in our test classes. I've also updated the test class to use the new classes.

Reply:
I've also converted the integration tests to JUnit 5 as discussed in the chat.